VCFv4.3.tex

```diff
@@ -326,132 +327,147 @@ \subsubsection{Fixed fields}
-\begin{itemize}
-  \item AA : ancestral allele
-  \item AC : allele count in genotypes, for each ALT allele, in the same order as listed
-  \item AD, ADF, ADR: read depths for each allele; total (AD), on the forward (ADF) and the reverse (ADR) strand (Integer, Number=R)
-  \item AF : allele frequency for each ALT allele in the same order as listed: use this when estimated from primary data, not called genotypes
-  \item AN : total number of alleles in called genotypes
-  \item BQ : RMS base quality at this position
-  \item CIGAR : cigar string describing how to align an alternate allele to the reference allele
-  \item DB : dbSNP membership
-  \item DP : combined depth across samples, e.g. DP=154
-  \item END : end position of the variant described in this record (for use with symbolic alleles)
-  \item H2 : membership in hapmap2
-  \item H3 : membership in hapmap3
-  \item MQ : RMS mapping quality, e.g. MQ=52
-  \item MQ0 : Number of MAPQ == 0 reads covering this record
-  \item NS : Number of samples with data
-  \item SB : strand bias at this position
-  \item SOMATIC : indicates that the record is a somatic mutation, for cancer genomics
-  \item VALIDATED : validated by follow-up experiment
-  \item 1000G : membership in 1000 Genomes
-  \item $\ldots$ see Section~\ref{sv-info-keys} for a list of INFO keys reserved for structural variants.
-\end{itemize}
+  INFO fields are encoded as a semicolon-separated series of short keys with optional values in the format: $<$key$>$=$<$data$>$[,data]. 
+  INFO keys must match the regular expression \texttt{\^{}[A-Za-z\_][0-9A-Za-z\_.]*\$}, duplicate fields are not allowed. Arbitrary keys are permitted, although the sub-fields listed in Table~\ref{table:reserved-info} are reserved (albeit optional).
+  
+  \begin{table}[htbp]
+    \centering
+    \begin{tabularx}{\textwidth}{ | l | l | l | X | }
+	Field		& Number	& Type		& Description \\ \hline
+	AA		& 1		& String	& Ancestral allele \\
+	AC		& A		& Integer	& Allele count in genotypes, for each ALT allele, in the same order as listed  \\
+	AD		& R		& Integer	& Total read depth for each allele \\
+	ADF		& R		& Integer	& Read depth for each allele on the forward strand \\
+	ADR		& R		& Integer	& Read depth for each allele on the reverse strand \\
+	AF		& A		& Float		& Allele frequency for each ALT allele in the same order as listed (use this when estimated from primary data, not called genotypes) \\
+	AN		& 1		& Integer	& Total number of alleles in called genotypes \\
+	BQ   		& 1		& Float		& RMS base quality \\
+	CIGAR		& A		& String	& Cigar string describing how to align an alternate allele to the reference allele \\
+	DB		& 0		& Flag		& dbSNP membership \\
+	DP		& 1		& Integer	& Combined depth across samples \\
+	END		& 1		& Integer	& End position (for use with symbolic alleles) \\
+	H2		& 0		& Flag		& HapMap2 membership \\
+	H3		& 0		& Flag		& HapMap3 membership \\
+	MQ		& 1		& Integer	& RMS mapping quality \\
```

Comment:
Both.

RMS is a floating point value, and also GATK emits a float rather than a
integer. See here, from a recent production BAM:
```

##INFO=<ID=MQ,Number=1,Type=Float,Description="RMS Mapping Quality">

```

On Mon, Jun 13, 2016 at 10:51 AM, John Marshall <notifications@github.com>
wrote:

> In VCFv4.3.tex
> <https://github.com/samtools/hts-specs/pull/123#discussion_r66805284>:
>
> > +	Field		& Number	& Type		& Description \\ \hline
> > +	AA		& 1		& String	& Ancestral allele \\
> > +	AC		& A		& Integer	& Allele count in genotypes, for each ALT allele, in the same order as listed  \\
> > +	AD		& R		& Integer	& Total read depth for each allele \\
> > +	ADF		& R		& Integer	& Read depth for each allele on the forward strand \\
> > +	ADR		& R		& Integer	& Read depth for each allele on the reverse strand \\
> > +	AF		& A		& Float		& Allele frequency for each ALT allele in the same order as listed (use this when estimated from primary data, not called genotypes) \\
> > +	AN		& 1		& Integer	& Total number of alleles in called genotypes \\
> > +	BQ   		& 1		& Float		& RMS base quality \\
> > +	CIGAR		& A		& String	& Cigar string describing how to align an alternate allele to the reference allele \\
> > +	DB		& 0		& Flag		& dbSNP membership \\
> > +	DP		& 1		& Integer	& Combined depth across samples \\
> > +	END		& 1		& Integer	& End position (for use with symbolic alleles) \\
> > +	H2		& 0		& Flag		& HapMap2 membership \\
> > +	H3		& 0		& Flag		& HapMap3 membership \\
> > +	MQ		& 1		& Integer	& RMS mapping quality \\
>
> Re *MQ (RMS)*, are you suggesting float based on what one would think an
> *RMS* value would be, or based on what you have seen in VCF files in the
> wild?
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/samtools/hts-specs/pull/123/files/e0976c64b96acc9671f5f993a5633bce0ae1f11a#r66805284>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACnk0pHdGW1cJOqlcny3BoKDGsZJ1NBaks5qLW5dgaJpZM4HF2zp>
> .
>